helpers.js

```diff
@@ -62,6 +62,40 @@ async function temporaryFileSentinel(fileDataArray, functionToWatch) {
   }
 }
 
+async function multipleTemporaryFileSentinels(fileContentsObject, functionToWatch) {
```

Comment:
1. It's still a single sentinel, with such difference that now it watches multiple files - `MultipleTemporaryFilesSentinel`.